citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -141,11 +145,22 @@ $(function() {
 		var authId = $(this).attr('data-authority-id');
 		$("#uriEditorLocalId").val(authId);
 		$("#editorAuthorityUsed").html("Using stored authority entry <i>" + $(this).attr('data-authority-name') + "</i>.");
+		showPersonNameInModal($(this).attr('data-authority-name'), "Editor");
 		$("#uriLoadingFoundEditor").popover('hide');
 		event.preventDefault();
 	});
 });
 
+function showPersonNameInModal(name, personType){
+	if(name.split(",")[0] != null && name.split(",")[0].length > 0) {
+		$("#lastName"+personType).val(name.split(",")[0]);
+	}
+	
+	if(name.split(",")[1] != null && name.split(",")[1].length > 0) {
+		$("#firstName"+personType).val(name.split(",")[1]);
+	}
+}
```

Comment:
if there is no comma in the name, let's split at blanks and assume the last part is the last name.

Reply:
if there is no comma in the name, then the last name is the last part of the name:
Einstein, Albert
Albert Einstein